### ISSUE TITLE
Regular css properties causes mismatch in rehydration on ssr

### ISSUE THREAD

**Issue by aga5tya** [rofrischmann/fela#705]
**Type**: Bug / Question

| Packages | Version |
| -------- | ------- |
| fela     | 10.2.4   |

-------------------
#### Description

Style function with keys as regular css properties (e.g. 'max-height') produces a mismatch during ssr and client rehydration.

#### Steps to reproduce
Reproduction on Codesandbox using nextjs

https://codesandbox.io/embed/91pn8r0z6w

#### Expected Behavior
Styles should be in sync and rehydrate from ssr state

#### Actual Behavior
Throws warning on load with below
```
Warning: Prop `className` did not match. Server: "a b c d" Client: "a b e f"
```

@rofrischmann  Any pointers or help on where this could be going wrong ?

**Comment by rofrischmann** [comment]
Hmm, why do you use lower-case properties at all?

**Comment by aga5tya** [comment]
Actually there are few legacy components in our codebase that have these and is huge to track all instances. I'm moving them as and when encountered, but was wondering if can be sorted at lib itself.

**Comment by rofrischmann** [comment]
well yeah, I can't really tell why that would be a problem in the first place, as we are transforming camelCase properties anyways...

**Comment by aga5tya** [comment]
Right now it's not much of a priority tho. I'l dig deeper by this weekend and figure out where it's going wrong.

**Comment by aga5tya** [comment]
> well yeah, I can't really tell why that would be a problem in the first place, as we are transforming camelCase properties anyways...

That indeed was the case. Was properly done during rehydration but not camelCased during client pass. Fixed at #707. Please review
